interface/grpc/utils/metadata.go

```diff
@@ -0,0 +1,6 @@
+package utils
```

Comment:
This should be in grpc package. Also:

[Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains.](https://blog.golang.org/package-names)

Reply:
It cannot be put in the grpc package because of import cycling issue..

Reply:
oh, ok, so at least let's change the name of package, maybe just `header`

grpc/header - list of headers + function to manipulate header (for example check values, status, cerating etc..)

Reply:
Moved to coreapi